modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLViewFileEntryDisplayContext.java

```diff
@@ -123,7 +123,8 @@ public String getDocumentTitle() throws PortalException {
 		}
 		else {
 			_documentTitle = StringBundler.concat(
-				fileVersion.getTitle(), StringPool.OPEN_PARENTHESIS,
+				fileVersion.getTitle(), StringPool.SPACE,
+				StringPool.OPEN_PARENTHESIS,
 				_language.get(_httpServletRequest, "version"),
 				fileVersion.getVersion(), StringPool.CLOSE_PARENTHESIS);
```

Comment:
I think we should also add an space before the version, otherwise you will see "Version1.0" instead of "Version 1.0". See screenshot in the LPS

Reply:
You're right. I'll that in a minute.